src/module/migration/migrations/937-remove-aura-class-traits.ts

```diff
@@ -0,0 +1,55 @@
+import { ItemSourcePF2e } from "@item/base/data/index.ts";
+import { MigrationBase } from "../base.ts";
+
+export class Migration937RemoveAuraClassTraits extends MigrationBase {
+    static override version = 0.937;
+
+    override async updateItem(source: ItemSourcePF2e): Promise<void> {
+        for (const rule of source.system.rules) {
+            if (
+                !rule ||
+                rule.key !== "Aura" ||
+                !("traits" in rule) ||
+                !Array.isArray(rule.traits) ||
+                rule.traits.length === 0
+            ) {
+                continue;
+            }
+
+            const newTraits = rule.traits.filter((t) => !classTraits.includes(t));
```

Comment:
Test for presence in effect traits, not for lack of presence in class traits (and import `effectTraits`).

Reply:
We need it to not KO homebrew effect traits. What you suggest might be a bit aggressive, especially if certain modules got disabled and they're updating to V13.